lib/find-emails.js

```diff
@@ -75,11 +82,18 @@ module.exports = function findEmails (args, callback) {
     port: args.port,
     tls: true,
     keepAlive: false,
-    // debug: function (info) { console.log(info) },
     tlsOptions: {
       secureProtocol: 'TLSv1_method'
     }
-  })
+  }
+
+  if (args.debug) {
+    imapOptions.debug = function (info) {
```

Comment:
You're not and you know it :stuck_out_tongue:
That's exactly why we have a review process.